examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,456 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# show_final_avg_density(result_full)\n",
+    "plot_avg_density_2D(get_avg_density(result_full), register)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f8623ff",
+   "metadata": {},
+   "source": [
+    "## Run AHS program in the blockade subspace\n",
+    "\n",
+    "As we can see above, it takes a long time to finish the simulation because the *full* Hamiltonian is of size $2^{15}\\times 2^{15}$. However, because of Rydberg blockade, if neighboring atoms are within each other's Rydberg blockade radius $R_b$, they are very unlikely to be excited to the Rydberg states simultaneously. Given that (see [notebook 00](https://github.com/aws/amazon-braket-examples/blob/main/examples/analog_hamiltonian_simulation/00_Introduction_of_Analog_Hamiltonian_Simulation_with_Rydberg_Atoms.ipynb))\n",
```

Comment:
I am not sure we should refer to it as notebook 00

Reply:
I change it to "this notebook". Please check.